akd/src/append_only_zks.rs

```diff
@@ -111,7 +113,7 @@ impl Azks {
     /// An azks is built both by the [crate::directory::Directory] and the
     /// auditor. However, both constructions have very minor differences, and
     /// the append_only_exclude_usage bool keeps track of this.
-    pub async fn batch_insert_leaves<S: Database + Sync + Send>(
+    pub async fn batch_insert_leaves<S: Database + Sync + Send + 'static>(
```

Comment:
Let's take the opportunity to add derivations to the base Database trait. The rewriting everywhere of sync + send is just getting silly lol. But maybe it makes sense to leave 'static here so the lifetime is only usage based.

Reply:
#321 🙂